### PR TITLE
Feature/gh 13916 ng 13 2

### PR DESCRIPTION
**Pull request by psmul**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -102,23 +102,23 @@
...
-    "@angular/platform-server": "^13.1.1",
+    "@angular/animations": "^13.2.0-rc.1",
+    "@angular/common": "^13.2.0-rc.1",
+    "@angular/compiler": "^13.2.0-rc.1",
+    "@angular/core": "^13.2.0-rc.1",
+    "@angular/forms": "^13.2.0-rc.1",
+    "@angular/platform-browser": "^13.2.0-rc.1",
+    "@angular/platform-browser-dynamic": "^13.2.0-rc.1",
+    "@angular/platform-server": "^13.2.0-rc.1",
     "@angular/pwa": "^13.0.1",
```
Comment (Platonn) [inline package.json]: let's bump it too, just to align versions

File: tsconfig.compodoc.json [hunk 2]
```diff
@@ -8,7 +8,7 @@
     "moduleResolution": "node",
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
-    "target": "es5",
+    "target": "es6",
```
Comment (Platonn) [inline tsconfig.compodoc.json]: nitpick: Was it deliberate to change compodoc tsconfig target in this PR to es6?

File: tsconfig.json [hunk 3]
```diff
@@ -8,7 +8,7 @@
     "declaration": false,
     "moduleResolution": "node",
     "experimentalDecorators": true,
-    "target": "es5",
+    "target": "es2015",
```
Comment (Platonn) [inline tsconfig.json]: nitpick: Was it deliberate to change compodoc tsconfig target in this PR to es2015?

File: package.json [hunk 4]
```diff
@@ -144,11 +144,11 @@
...
     "@angular-eslint/schematics": "^13.0.1",
     "@angular-eslint/template-parser": "^13.0.1",
-    "@angular/cli": "^13.1.2",
+    "@angular/cli": "^13.2.0-rc.1",
     "@angular/compiler-cli": "^13.1.1",
     "@angular/language-service": "^13.1.1",
     "@ngrx/store-devtools": "^13.0.2",
-    "@nguniversal/builders": "^13.0.0",
+    "@nguniversal/builders": "^13.1.0-next.1",
     "@schematics/angular": "^13.0.1",
```
Comment (Platonn) [inline package.json]: 1. Let's bump them all, to align versions.

2. Please also bump those guys (I couldn't add a comment for those lines this PR):
```
    "@angular-builders/custom-webpack": "^13.0.0",
    "@angular-devkit/build-angular": "^13.1.2",
    "@angular-devkit/core": "^13.1.2",
``` 

3. Let's also bump the "peerDependencies" in all libraries' package.jsons, e.g. (`@angular/core` in `projects/core/package.json` etc.)

4. In the end, let's align the bumped versions also in the `dependencies.json`

File: tsconfig.json (hunk 3 above)
Comment (psmul) [reply tsconfig.json]: I might have rushed things. I think this still should be done, but maybe in a separate commit on the main branch, right?

File: package.json (hunk 4 above)
Comment (psmul) [reply package.json]: `@nguniversal/builders` haven't received any release for 12 days, so we're up to date here
`@angular-builders/custom-webpack` also haven't received a stable release, only 13.1.0-beta available

**Review by Platonn — APPROVED** [review]

**Pull request merged**